plugins/modules/ibm_resource_group.py

```diff
@@ -13,8 +13,8 @@
 ---
 module: ibm_resource_group
 short_description: Manage C(resource_groups) for Resource Manager.
-author: Kavya Handadi (@kavya498)
-version_added: "0.0.1-beta0"
+author: Umar Ali (@umarali-nagoor)
+version_added: "1.0.0"
```

Comment:
The `version_added` value should remain at the version that the module was added. Same for the rest of the modules.

Reply:
@umarali-nagoor Can you check on this